readme.md

```diff
@@ -105,13 +105,42 @@ The following data attributes are supported:
    the configuration values
 - `data-url`: (required) The K-Link compatible endpoint to use for searching
 - `data-token`: (required) The API token to obtain access to the Search
-- `data-display`: The display style.
- - overlay the search box is visible and can expand on top of the other elements of the page when active. The
+- `data-display`: The display style:
+  + `overlay` the search box is visible and can expand on top of the other elements of the page when active. The
    results are presented in a dialog below the search input field
- - embed the search box and the results are visible in a page area and cannot hide other elements already in the page
+  + `embed` the search box and the results are visible in a page area and cannot hide other elements already in the page
 - `data-collapsed`: If the search form should be collapsed to use less space on the page, for example if used in the header.
   This is a boolean attribute, so no explicit value is required.
 
+## <a name = 'token'>How to get a token</a>
+
+Go to the registry page of the K-Link network you want to connect to (typically https://public.your_k-link_network/registry) and create an account. If you don't know the address of the registry, contact the technical counterpart of the K-Link network you want to connect to or check if this K-Link network is referenced [below](#list-k-links).
+
+Go to Applications and click on `Add`
+
+Give a name to the application on which the K-Search client will run, enter the domain of the app ("mywebsite.com", "mycompany.de", etc.) and choose the permissions "data-search" and "data-view"
+
+Put the status on `enabled`
+
+Click on `Save`, your token has been generated
+
+## <a name = 'endpoint'>How to get the K-Link endpoint url</a>
+
+Contact the technical counterpart of the K-Link network you want to connect to or check if this K-Link network is referenced [below](#list-k-links).
+
+## <a name = 'list-k-links'>Non exhaustive list of publicly available K-Link networks</a>
```

Comment:
@shenriod I think is better to point to https://k-link.technology/network.html

In this way the library remains generic and the website do the work of presenting the list of networks and their parameters.

Reply:
I have no big problem with your suggestion but:

1) My understanding was that k-link.technology was not yet sufficiently finalized and that it should not be publicly used yet?
2) k-link.technology does not indicate the URLs of the endpoints and of the registry pages. Is that an information that will be added? The idea is that web-developers should easily find all the info they need to use the k-search client, without having to "bother" anyone for information
3) What I like about this list on GitHub is that anyone could propose a PR to add other openly available K-Link networks, without "bothering" OOT to ask an update on the website. In the future, there hopefully also will be new K-Link networks that have nothing to do neither with GIZ nor with OOT

Reply:
> My understanding was that k-link.technology was not yet sufficiently finalized and that it should not be publicly used yet?

I think it can be used, and in fact we did for the workshop. Even if is a pretty basic website, it contains the minimum required information

> k-link.technology does not indicate the URLs of the endpoints and of the registry pages. Is that an information that will be added? The idea is that web-developers should easily find all the info they need to use the k-search client, without having to "bother" anyone for information

This can be solved by adding it to the website. We want k-link.technology to be the reference location for the technology. If we integrate the listing of the network endpoint in the connection library, we should do it for every specific implementation of that library, e.g. the php, Java, C# ones

> What I like about this list on GitHub is that anyone could propose a PR to add other openly available K-Link networks, without "bothering" OOT to ask an update on the website. In the future, there hopefully also will be new K-Link networks that have nothing to do neither with GIZ nor with OOT

the pull to add "users" is a general approach, but consider that is normally done on the repository of the website or on the main repository of the technology. A list of "users" make more sense on the K-Link repository than on the javascript implementation of the client.

Reply:
Fine, agreed!

I will make the changes but we cannot accept the PR before the information about tokens and endpoints is on the website, I guess.